src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -103,10 +116,21 @@ def current_tables(self, tables: TablesCrawler, workspace_name: str, catalog_nam
         for table in tables_snapshot:
             yield Rule.initial(workspace_name, catalog_name, table, self._recon_tolerance_percent)
 
+    @staticmethod
+    def tables_not_mapped(tables_crawler: TablesCrawler, current_tables: list[Rule], workspace_name: str):
```

Comment:
make private

Reply:
The implementation approach has been changed.